src/types/config.d.ts

```diff
@@ -35,10 +35,16 @@ export interface ConfigOptions {
   storage_type?: 'cookies' | 'localStorage'
   headers?: { [key: string]: string }
   disableCart?: Boolean
-  reauth?: Boolean,
+  reauth?: Boolean
```

Comment:
Thank you @samblacklock, `reauth` and `disableCart` values are the same as before and I have not updated them. I also had the thought to change its value but was not sure what's the usage of `Boolean` for those. Let me know if both the values should be updated with `boolean` rather than `Boolean`

Reply:
Thanks @krish4uu if you could change both that would be great (I know it wasn't your work but it makes sense to do it while you are updating this ConfigOptions `type` :))

Reply:
Hi @samblacklock, sure I'll update it